packages/react-composites/tests/browser/chat/hermetic/FileSharing.test.ts

```diff
@@ -356,7 +356,7 @@ test.describe('Filesharing Edit Message', async () => {
     await page.waitForSelector(dataUiId('file-download-card-group'));
     await page.locator(dataUiId('chat-composite-message')).click();
     await page.locator(dataUiId('chat-composite-message-action-icon')).click();
-    await page.waitForSelector('[id="chat-composite-message-contextual-menu"]');
+    await page.waitForSelector(dataUiId('chat-composite-message-contextual-menu-edit-action'));
     await page.locator(dataUiId('chat-composite-message-contextual-menu-edit-action')).click();
     await page.waitForSelector('[id="editbox"]');
```

Comment:
Should  this edit box still be an id? I notice everything else is a dataUiId now?

Reply:
Good catch, updated!